fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.camel.serialization;
+
+import static org.apache.camel.LoggingLevel.INFO;
+import static org.apache.camel.LoggingLevel.DEBUG;
+import static org.apache.camel.Exchange.FILE_NAME;
+import static org.apache.camel.builder.PredicateBuilder.not;
+import static org.apache.camel.builder.PredicateBuilder.or;
+import static org.apache.camel.component.exec.ExecBinding.EXEC_COMMAND_ARGS;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_IDENTIFIER;
+import static org.fcrepo.camel.JmsHeaders.EVENT_TYPE;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.builder.xml.Namespaces;
+import org.fcrepo.camel.RdfNamespaces;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * A router for serializing fedora objects and binaries.
+ *
+ * @author Bethany Seeger
+ * @since 2015-09-28
+ */
+
+public class SerializationRouter extends RouteBuilder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SerializationRouter.class);
+
+    private static final String isBinaryResourceXPath =
+        "/rdf:RDF/rdf:Description/rdf:type[@rdf:resource='" + RdfNamespaces.REPOSITORY + "Binary']";
+    /**
+     * Configure the message route workflow
+     *
+     */
+
+    public void configure() throws Exception {
+
+        final Namespaces ns = new Namespaces("rdf", RdfNamespaces.RDF);
```

Comment:
`Namespaces` is "fluent" and you can use `static import`, so just `final Namespaces ns = new Namespaces("rdf", RDF).add("fedora", REPOSITORY);`